docs/sources/investigate/index.md

```diff
@@ -34,7 +36,7 @@ When you use Explore Profiles, your investigations follow these steps.
    - From the Labels view: Select **Add to filters** from one of the areas of interest.
 1. Optional: Click and drag on any graph to select a specific time frame or data range.
 1. Use the **Labels** view to select two processes to compare.
-   - Select the checkbox in two graphs. Select **Compare** to view your selections in the **Comparison diff view**.
-   - Use the **Service** and **Profile** drop-downs to change the service or profile type displayed.
+   - Select the checkbox in two graphs. Select **Compare** to view your selections in the **Diff flame graph** view.
+   - Use the **Service** and **Profile type** drop-downs to change the service or profile type displayed.
```

Comment:
```suggestion
   - Select both a time range in the baseline graph and in the comparison graph to view the diff flame graph.
```

Reply:
Thank you for this update

Reply:
Same here: is this suggestion correct?

Reply:
Resolving since we discussed this in Slack.